crates/aptos/CHANGELOG.md

```diff
@@ -5,6 +5,7 @@ All notable changes to the Aptos CLI will be captured in this file. This project
 ## Unreleased
 - Determine network from URL to make explorer links better for legacy users
 - Add support for AIP-80 compliant strings when importing using the CLI arguments or manual input.
+- Add option `--print-metadata` to `aptos move decompile` and `aptos move disassemble` to print out the metadata attached to the bytecode.
```

Comment:
Update with the new flag.

Reply:
done

Reply:
I'm wondering why we are using a flag to repurpose `decompile` and `disassemble` for this functionality, instead of adding a new command?  It seems (1) redundant to have it for both of them, and (2) strange to disable the primary functionality of the command if this flag is enabled.

Reply:
This feature is using `mv` files as input, so it is natural to add it to `decompile` and `disassemble` command and I don't think it is worth a standalone command. Also, redundancy comes from the fact that both commands share the same BytecodeCommand.